README.md

```diff
@@ -86,9 +70,38 @@ shapely
 fiona
 ```
 
-One of dependencies of `eo-learn-mask` subpackage is `lightgbm` package. On Windows it requires 64 bit Python distribution. If having problems during installation please check [LightGBM installation guide](https://lightgbm.readthedocs.io/en/latest/Installation-Guide.html).
+One of the `[MASK_EXTRA]` dependencies is the `lightgbm` package. On Windows it requires 64 bit Python distribution. If having problems during installation please check [LightGBM installation guide](https://lightgbm.readthedocs.io/en/latest/Installation-Guide.html).
+
+### PyPi distribution
+
+The package requires Python version **>=3.8** . If this is your first time installing `eo-learn`, or you aren't sure what kind of installation you require, than feel free to install it with:
 
-Some subpackages contain extension modules under `extra` subfolder. Those modules typically require additional package dependencies which don't get installed by default.
+```bash
+pip install "eo-learn[ALL]"
+```
+
+This will install the package and all the default modules. Otherwise, if you're interested in a super light, core-only installation of `eo-learn`, then install it via:
+
+```bash
+pip install eo-learn
+# pip install "eo-learn[CORE]"  # this does the same thing
+```
+
+In order to avoid heavy environments, it is possible to install various scopes of package dependencies. Dependencies for each of the subfolders above (or their extra dependencies) can be installed via e.g.:
+
+```bash
+pip install "eo-learn[FEATURES]"
+pip install "eo-learn[FEATURES_EXTRA]"
```

Comment:
I would perhaps structure this differently. More in a way of:
"To better control your dependencies you can install just the core dependencies of `eolearn` with `pip install eo-learn` or you can customize, which modules you want operational by providing the corresponding extras, e.g. `pip install eo-learn[IO, MASK]` if you are only interested in the `io` and `mask` submodules.

And then i'd also have a list of all these subpackage extras and their extra extras? I feel like this is the place to have this information, we shouldn't expect users to rummage through our `pyproject.toml`

Reply:
updated and added the list